ReadMe.md

```diff
@@ -1,3 +1,11 @@
+# Git & GitHub Practical Guide
+
+#### Introduction
```

Comment:
This is level four heading, please reduce this to level 2 and add another title.